test/system/120-load.bats

```diff
@@ -147,4 +147,43 @@ verify_iid_and_name() {
        "Diagnostic from 'podman load' without redirection or -i"
 }
 
+@test "podman load - multi-image archive" {
+    img1="quay.io/libpod/testimage:00000000"
+    img2="quay.io/libpod/testimage:20200902"
+    archive=$PODMAN_TMPDIR/myimage-$(random_string 8).tar
+
+    run_podman pull $img1
+    run_podman pull $img2
+
+    run_podman save -m -o $archive $img1 $img2
+    run_podman rmi -af
+    run_podman load -i $archive
+
+    run_podman image exists $img1
+    run_podman image exists $img2
```

Comment:
Please `rmi` anything you bring in. Sorry, I should've caught that earlier.

Reply:
Thanks, fixed now :)